languages/JavaScript/exercises/concept/arrays-basic/.docs/hints.md

```diff
@@ -0,0 +1,23 @@
+### 1. Retrieve a card from a stack
+
+- Note that the array index starts from `0`.
+
+### 2. Exchange a card in the stack
+
+- The array is a mutable structure, you can change its content anytime.
+
+### 3. Create a stack of cards
+
+- The `push` method adds a new value at the end of the array. *e.g.* `array.push(newValue)`
```

Comment:
For our hints we'd like them to not give away the solution literally, but instead prefer to link to an external resource. Here is an [example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/hints.md) what that could look like. This same comments also applies to the three tasks following this task.